### PR TITLE
Task/calm 21376 calm vm migration ahv vmware

### PR DESCRIPTION
**Pull request by kumarsarath588**

### CODE REVIEW

File: calm-integrations/ey-calm-migration-scripts/ey-vmware-post-migration-script.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+    NSE = model.SubstrateElement.query(instance_id=instance_id)
+    if NSE:
+        NSE = NSE[0]
+        if NSE.spec.resources.account_uuid != DEST_ACCOUNT_UUID:
+            log.info("Updating VM substrate for '{}' with instance_id '{}'.".format(vm_name, instance_id))
+            NSE.spec.resources.account_uuid = DEST_ACCOUNT_UUID
+            NSE.spec.datastore = datastore
+            NSE.spec.host = host_uuid
+            #NSE.platform_data = json.dumps(vm)
```
Comment (calmtanmay) [inline calm-integrations/ey-calm-migration-scripts/ey-vmware-post-migration-script.py:78]: We need to take opinion on this.

File: calm-integrations/ey-calm-migration-scripts/ey-vmware-post-migration-script.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+            #for i in range(len(NSE.spec.resources.nic_list)):
+                #NSE.spec.resources.nic_list[i].net_name = "this should be net_name"
+                #NSE.spec.resources.nic_list[i].nic_type = "this should be nic_type"
```
Comment (calmtanmay) [inline calm-integrations/ey-calm-migration-scripts/ey-vmware-post-migration-script.py:81]: We need to write special logic for these, similar to this golang code- https://github.com/ideadevice/calm-epsilon-shared/blob/master/src/ces/server/indra/provider/vmware/vmware_network_util.go#L137 and https://github.com/ideadevice/calm-epsilon-shared/blob/master/src/ces/server/indra/provider/vmware/vmware.go#L2316

File: calm-integrations/ey-calm-migration-scripts/ey-vmware-post-migration-script.py [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+            for i in range(len(NSE.spec.resources.disk_list)):
+                if NSE.spec.resources.disk_list[i].disk_type == "disk":
```
Comment (calmtanmay) [inline calm-integrations/ey-calm-migration-scripts/ey-vmware-post-migration-script.py:83]: Are all this disk in EY environment is of type `disk`?

File: calm-integrations/ey-calm-migration-scripts/helper.py [hunk 4]
```diff
@@ -0,0 +1,337 @@
...
+    vm_uuids = []
+    for deploy in app.active_app_profile_instance.deployments:
+        for de in deploy.elements:
+            sub_el = de.substrate_element
+            if sub_el.type == "VMWARE_VM":
+                vm_uuids.append(str(sub_el.instance_id))
+    # Change ownership of all vm's to New project
+    # Same step mentioned for app need to follow for vm
+    for vm_uuid in vm_uuids:
+        handle_entity_project_change("vm", vm_uuid, tenant_uuid, new_project_name, new_project_uuid)
```
Comment (calmtanmay) [inline calm-integrations/ey-calm-migration-scripts/helper.py:237]: Are we changing project of vmware VMs?

**Review by AmitVasantSalunke — APPROVED** [review]
LGTM

**Pull request merged**